src/controllers/v1/sessions.js

```diff
@@ -154,7 +154,11 @@ module.exports = class Sessions {
 				req.params.id,
 				req.decodedToken,
 				req.headers['timezone'],
-				isAMentor(req.decodedToken.roles)
+				isAMentor(req.decodedToken.roles),
+				true,
```

Comment:
@nevil-mathew can you set isSelfEnrolled = true and then pass it
and set  sessions= {}